pop-sandbox/README.md

```diff
@@ -0,0 +1,60 @@
+# Pop Sandbox
+
+Implementation of the [`pop_drink::Sandbox`](https://github.com/r0gue-io/pop-drink) struct for the Pop Network runtimes (located in `pop-node/runtime`) required for the quasi testing with `drink`.
+
+In the context of quasi-testing with pop-drink, a sandbox refers to an isolated runtime environment that simulates the behavior of a full node, without requiring an actual node. It can emulate key processes (where runtime `pallets` are involved) such as block initialization, execution, and block finalization.
+
+## Getting Started
+
+### Installation
+
+```toml
+pop_drink = { version = "1.0.0",  package = "pop-drink" }
+```
+
+### Import Sandbox for the specific runtime
+
+- For `devnet` runtime
+
+Implementation of the sandbox runtime environment for `devnet` runtime located in `pop-node/runtime/devnet`
+
+```rs
+use pop_sandbox::DevnetSandbox;
+```
+
+- For `testnet` runtime
+
+Implementation of the sandbox runtime environment for `testnet` runtime located in `pop-node/runtime/testnet`
+
+```rs
+use pop_sandbox::TestnetSandbox;
+```
+
+- For `mainnet` runtime
```

Comment:
Can we remove this. I don't want people to ask about a mainnet if we don't have it yet :)